src/ray/raylet/scheduling/scheduler_resource_reporter.cc

```diff
@@ -69,6 +69,11 @@ void SchedulerResourceReporter::FillResourceUsage(
   absl::flat_hash_set<SchedulingClass> visited;
   auto fill_resource_usage_helper = [&](const auto &range, bool is_infeasible) mutable {
     for (auto [scheduling_class, count] : range) {
+      if (count == 0) {
```

Comment:
cc @iycheng  - this is okay right? I was curious why we didn't skip those scheduling class with count == 0?

Reply:
ah, actually, backlog are passed with count set to 0 explicitly.

Reply:
So my change is not correct.